tsconfig.json

```diff
@@ -16,10 +16,13 @@
       // this is the package name we use (in the interface imports, --package for generators) */
       "polymesh-subql/*": ["src/*"],
       // here we replace the @polkadot/api augmentation with our own, generated from chain
-      "@polkadot/api/augment": ["src/api-interfaces/augment-api.ts"],
+      // "@polkadot/api/augment": ["src/api-interfaces/augment-api.ts"],
```

Comment:
Do we not need these anymore?

Reply:
I don't think those were used earlier as well. We can remove those for now